pkgs/development/libraries/gpgme/default.nix

```diff
@@ -24,20 +16,17 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkgconfig gnupg ];
 
   configureFlags = [
-    "--enable-fixed-path=${gpgStorePath}/bin"
+    "--enable-fixed-path=${gnupg}/bin"
   ];
 
   NIX_CFLAGS_COMPILE =
     with stdenv; lib.optional (system == "i686-linux") "-D_FILE_OFFSET_BITS=64";
 
-  AM_CXXFLAGS =
-    with stdenv; lib.optional (isDarwin) "-D_POSIX_C_SOURCE=200809L";
-
```

Comment:
That should be fixed in 1.9.0: https://bugs.gnupg.org/gnupg/issue2910

Thanks @orivej for noticing this.